sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -462,3 +462,12 @@ INSERT INTO list_options (list_id,option_id,title) VALUES ('lists','apps','Apps'
 INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUES ('apps','*OpenEMR','main/main_screen.php',10,1,0);
 INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity) VALUES ('apps','Calendar','main/calendar/index.php',20,0,0);
 #EndIf
+
+#IfNotTable patient_birthday_alert
+CREATE TABLE `patient_birthday_alert` (
+  `pid` bigint(20) NOT NULL,
+  `user_id` bigint(20) NOT NULL,
```

Comment:
for me:
likely need to default these to 0 if not allow NULL (it should never happen, but then at least predictable if it does)